### PR TITLE
Issue-413- Slider Image Scheduling

### PR DESCRIPTION
**Pull request by SunilParisi123**
Resolves #issueNumber  
Type: **feature|bugfix|**

## Issue
Description of the issue this PR is solving, why it's happening, and how to reproduce it.

## Solution
Summarize your solution to the problem. Please include short description.

## Breaking changes
If you have a breaking changes, list them here, otherwise list none.

## Testing
1. List the steps needed for testing your PR.
2. Assume that everyone already know how to run the GrandNode, and do the basic configuration.
3. Be detailed enough that someone can work through it easily.

### CODE REVIEW

**Comment by SunilParisi123** [comment]
@KrzysztofPajak Can this be added as a feature extension for Slider images. https://github.com/grandnode/grandnode2/issues/413

Any suggestions appreciated.

**Comment by SunilParisi123** [comment]
@KrzysztofPajak  Any comments on this?

**Comment by SunilParisi123** [comment]
Could you please check it.

File: src/Plugins/Widgets.Slider/Extensions.cs [hunk 1]
```diff
@@ -61,5 +68,44 @@ public static bool HasProperty(this Type obj, string propertyName)
         }
     }
 
+    public static class DateTimeMappingExtensions
```
[the remark this answers is not in the document]
Comment (SunilParisi123) [reply src/Plugins/Widgets.Slider/Extensions.cs:71]: ok, will move it.

[the remark this answers is not in the document]
Comment (SunilParisi123) [reply src/Plugins/Widgets.Slider/Extensions.cs]: @KrzysztofPajak Moved the date time extension to Web.Common project as suggested and updated PR. Please check now.

[the remark this answers is not in the document]
Comment (KrzysztofPajak) [reply src/Plugins/Widgets.Slider/Extensions.cs]: @SunilParisi123 but I think this extensions should be removed from Grand.Web.Admin project

[the remark this answers is not in the document]
Comment (SunilParisi123) [reply src/Plugins/Widgets.Slider/Extensions.cs]: @KrzysztofPajak i have removed it from Grand.Web.Admin.Extensions.Mapping. Also referred from Grand.Web.Common wherever required.

File: src/Web/Grand.Web.Common/Extensions/DateTimeMappingExtensions.cs [hunk 2]
```diff
@@ -1,12 +1,16 @@
 ﻿using Grand.Business.Core.Interfaces.Common.Directory;
 
-namespace Grand.Web.Admin.Extensions.Mapping
+namespace Grand.Web.Common.Extensions
 {
     public static class DateTimeMappingExtensions
     {
         public static DateTime? ConvertToUserTime(this DateTime? datetime, IDateTimeService dateTimeService)
         {
-            if (datetime is { Kind: DateTimeKind.Utc }) datetime = dateTimeService.ConvertToUserTime(datetime.Value, TimeZoneInfo.Utc, dateTimeService.CurrentTimeZone);
```
Comment (KrzysztofPajak) [inline src/Web/Grand.Web.Common/Extensions/DateTimeMappingExtensions.cs:9]: Please return back this notation:
if (datetime is { Kind: DateTimeKind.Utc })

Comment (SunilParisi123) [reply src/Web/Grand.Web.Common/Extensions/DateTimeMappingExtensions.cs]: Done.

Comment (SunilParisi123) [reply src/Web/Grand.Web.Common/Extensions/DateTimeMappingExtensions.cs]: Can you please check now.

**Comment by KrzysztofPajak** [comment]
@SunilParisi123 Thanks for contributing

**Pull request merged**